packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -45,9 +46,13 @@ interface ComboboxContentProps {
 }
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
-  const { open, setOpen, wrapperRef } = React.useContext(ComboboxContext);
+  const { open, setOpen, wrapperRef, multiselect } =
+    React.useContext(ComboboxContext);
   const optionsExist = props.options.length > 0;
 
+  // Calculate the count of selected options
+  const selectedCount = props.selected ? 1 : 0;
```

Comment:
I'm assuming this is temporary code

a note for later, we'll need to make this look at the `selectedOptions` wrapper that will either be the external state passed in, or the internal state if we have the `onClose` callback

Reply:
Correct - just wanted something to actually change the label within the Header.  Which also means the Header component itself has temporary code based on just 1 selection.